.github/workflows/inspektor-gadget.yml

```diff
@@ -20,21 +22,17 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v1
 
-    - name: Build gadget container and publish to Registry
-      id: publish-registry
-      uses: elgohr/Publish-Docker-Github-Action@2.8
-      with:
-        name: ${{ secrets.CONTAINER_REPO }}
-        username: ${{ secrets.CONTAINER_REGISTRY_USERNAME }}
-        password: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
-        dockerfile: gadget.Dockerfile
-        snapshot: true
-        cache: ${{ github.event_name != 'schedule' }}
+    - name: Set IMAGE_TAG
+      run: |
+        TMP1=${GITHUB_REF#*/}
+        TMP2=${TMP1#*/}
+        echo IMAGE_TAG=${TMP2//\//-} >> $GITHUB_ENV
+
+    - name: Build gadget container
+      run: |
+        make gadget-container
 
     - name: Build kubectl gadget Plugin
-      env:
-        CONTAINER_REPO: ${{ secrets.CONTAINER_REPO }}
```

Comment:
Why removing `$CONTAINER_REPO`?

I think we should keep it, so that GitHub forks of IG could configure the GitHub action to build `kubectl-gadget` configured with their own container registry.

```
$ kubectl gadget deploy|grep image:
        image: docker.io/kinvolk/gadget:xxxxx
```

(ditto below)

Reply:
> Why removing `$CONTAINER_REPO`?

It's already defined in L13 and it's available to all steps of the build job. 
 

> ```
> $ kubectl gadget deploy|grep image:
>         image: docker.io/kinvolk/gadget:xxxxx
> ```

I'm not sure what you mean by this... Is it an error you hit or are you just trying to show that a user should be able to configure its own container registry?

Reply:
Oh, i didn't see. I meant the latter.